calculation/gmhazard_calc/gmhazard_calc/verification/GMM/psha_im_rrup_plot.py

```diff
@@ -164,16 +164,14 @@ def get_computed_gmms(
                                     result = empirical_factory.compute_gmm(
                                         fault, site, empirical_factory.GMM[model], im,
                                     )
-                                    # result is always tuple
-                                    # For Meta
-                                    # result_dict[tect_type][im][vs30][mag][model].append(
-                                    #     result[0][0]
-                                    #     if isinstance(result, Iterable)
-                                    #     else result[0]
-                                    # )
+                                    # For Meta - it is a tuple inside a list
                                     result_dict[tect_type][im][vs30][mag][model].append(
-                                        result[0]
+                                        result[0][0]
                                     )
+                                    # For non-Meta - result is always tuple
+                                    # result_dict[tect_type][im][vs30][mag][model].append(
+                                    #     result[0]
+                                    # )
```

Comment:
Couldn't you just leave the conditional in there then? and have it handle it automagically?